src/node.h

```diff
@@ -246,9 +246,10 @@ inline void NODE_SET_PROTOTYPE_METHOD(v8::Handle<v8::FunctionTemplate> recv,
 #define NODE_SET_PROTOTYPE_METHOD node::NODE_SET_PROTOTYPE_METHOD
 
 enum encoding {ASCII, UTF8, BASE64, UCS2, BINARY, HEX, BUFFER};
-enum encoding ParseEncoding(v8::Isolate* isolate,
-                            v8::Handle<v8::Value> encoding_v,
-                            enum encoding default_encoding = BINARY);
+NODE_EXTERN enum encoding ParseEncoding(v8::Isolate* isolate,
```

Comment:
Please move `v8::Isolate*` and friends on a next line at 4 space indent. No need to cascade things ;)